packages/frontend/src/router.definition.ts

```diff
@@ -57,18 +76,14 @@ const routes: RouteDef[] = [{
 		path: '/avatar-decoration',
 		name: 'avatarDecoration',
 		component: page(() => import('@/pages/settings/avatar-decoration.vue')),
-	}, {
-		path: '/roles',
-		name: 'roles',
-		component: page(() => import('@/pages/settings/roles.vue')),
 	}, {
 		path: '/privacy',
 		name: 'privacy',
 		component: page(() => import('@/pages/settings/privacy.vue')),
 	}, {
-		path: '/emoji-picker',
-		name: 'emojiPicker',
-		component: page(() => import('@/pages/settings/emoji-picker.vue')),
+		path: '/emoji-palette',
+		name: 'emoji-palette',
+		component: page(() => import('@/pages/settings/emoji-palette.vue')),
```

Comment:
ブラウザがurl覚えてくれてることも多いので、同じ用途のページなのでリンクするべきだと思います

```suggestion
		path: '/emoji-picker',
		redirect: '/settings/emoji-palette',
	}, {
		path: '/emoji-palette',
		name: 'emoji-palette',
		component: page(() => import('@/pages/settings/emoji-palette.vue')),
```

Reply:
設定のリダイレクトについては簡潔さを優先してこのままにしておきます